cmd/scollector/main.go

```diff
@@ -149,7 +150,7 @@ func readConf() {
 
 func main() {
 	flag.Parse()
-	if *flagPrint || *flagDebug {
+	if *flagPrint || *flagDebug || *flagHTTP {
 		slog.Set(&slog.StdLog{Log: log.New(os.Stdout, "", log.LstdFlags)})
```

Comment:
Thanks for such a quick review. 

The changes made above were influenced by inability to make respective flags configurable through configuration file instead of arguments. I faced this issue with both frequency and tcp/http switch and it is reflected by the code. 

If you could let me know, the way to do so , I will make those changes in our code base as they are desired.

Meanwhile, changes would be made for:

1. restore `freq` default to `15`.
1. make `http` as default way to send metrics. 


`*flagHTTP` is a mistake on my part, I will fix it.